crates/topos/src/components/node/commands/init.rs

```diff
@@ -14,8 +15,8 @@ pub struct Init {
     pub role: NodeRole,
 
     /// Subnet of your node
-    #[arg(long, env = "TOPOS_NODE_SUBNET", default_value = "topos")]
-    pub subnet: Option<String>,
+    #[arg(long, env = "TOPOS_NODE_SUBNET_ID", default_value = "topos")]
+    pub subnet_id: Option<String>,
```

Comment:
Not really the subnet-id though, here that's the subnet alias that points to the genesis.

It can be changed to subnet-id when it'll effectively take the subnet-id (the Sequencer public key for now, and eventually the ICE-FROST public key)

Reply:
Should we change it in the `base` config than as well? I was just confused as to why we pass the argument as `subnet` but in the config its `subnet_id`:

```
#[serde(default = "default_subnet")]
pub subnet_id: String,
```